src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -619,57 +678,105 @@ class Config:
+        during the initial domain join.
+
+        `kerberos_realm` in which the server is located. This parameter will be
+        automatically populated during the initial domain join. If the NAS has
+        an AD site configured and that site has multiple kerberos servers, then
+        the kerberos realm will be automatically updated with a site-specific
+        configuration to use those servers. Auto-detection is only performed
+        during initial domain join.
+
+        `kerberos_principal` kerberos principal to use for AD-related
+        operations outside of Samba. After intial domain join, this field will
+        be updated with the kerberos principal associated with the AD machine
+        account for the NAS.
+
+        `nss_info` controls how Winbind retrieves Name Service Information to
+        construct a user's home directory and login shell. This parameter
+        is only effective if the Active Directory Domain Controller supports
+        the Microsoft Services for Unix (SFU) LDAP schema.
+
+        `timeout` timeout value for winbind-related operations. This value may
+        need to be increased in  environments with high latencies for
+        communications with domain controllers or a large number of domain
+        controllers. Lowering the value may cause status checks to fail.
+
+        `dns_timeout` timeout value for DNS queries during the initial domain
+        join. This value is also set as the NETWORK_TIMEOUT in the ldap config
+        file.
+
+        `ldap_sasl_wrapping` defines whether ldap traffic will be signed or
+        signed and encrypted (sealed). LDAP traffic that does not originate
+        from Samba defaults to using GSSAPI signing unless it is tunnelled
+        over LDAPs.
+
+        `createcomputer` Active Directory Organizational Unit in which to
+        create the NAS computer object during domain join.
+
+        If blank, then the default OU is used during computer account creation.
+        Precreate the computer account in a specific OU. The OU string read from
+        top to bottom without RDNs and delimited by a "/". For example,
+        "createcomputer=Computers/Servers/Unix. Backslashes "\" are used as an
+        escape character, and not as a separator.
+
+        `idmap_backend` provides a plugin interface for Winbind to use varying
+        backends to store SID/uid/gid mapping tables. The correct setting
+        depends on the environment in which the NAS is deployed.
+
+        The Active Directory service will be started after a configuration
+        update if the service was initially disabled, and the updated
+        configuration sets `enable` to `True`. Likewise, the Active Directory
+        service will be stopped if `enable` is changed to `False`. If the
+        configuration is updated, but the initial `enable` state is `True`, and
+        remains unchanged, then the samba server will only be restarted.
```

Comment:
```suggestion
        remains unchanged, then only the samba server is restarted.
```

Reply:
That changes the meaning of the statement. I will remove its 'will', but will maintain the same meaning.